libraries/mapping/engines/cesium/src/lib/utils/cesiumAnimations.ts

```diff
@@ -60,7 +63,9 @@ export function animateInterpolateHeadingPitchRange(
     // Interpolate heading and pitch over time
     const currentHeading = CeMath.lerp(initialHeading, heading, easing(t));
     const currentPitch = CeMath.lerp(initialPitch, pitch, easing(t));
-    const currentRange = CeMath.lerp(initialRange, range, easing(t));
+    const currentRange = useCurrentDistance
```

Comment:
now skipping restauration of old range by default, should only care about pitch and heading.